### PR TITLE
Streamline the installation and distribution of the mmsdk package.

### PR DESCRIPTION
**Pull request by CristoJV**
The goal of this pull request is to streamline the installation and distribution of the ``mmsdk`` package. This update brings several beneficial changes:

1. The addition of a ``.gitignore`` file. This file helps maintain a clean code base by negating the need for the error-prone 'clean.sh' script before committing changes.

2. A comprehensive ``requirements.txt`` file that lists all necessary dependencies, including 'colorama' which was previously not mentioned in the README.md file.

3. A ``setup.py`` file to facilitate an easy and clean installation of the package in the Python environment. This approach is more user-friendly compared to modifying the ``PYTHON_PATH`` variable, which is generally not advised. The ``README.md`` file has been updated to reflect this improved installation process.

There are still a few minor adjustments to be made, particularly regarding the package's metadata. This pull request is currently a draft, allowing authors the opportunity to make further contributions, such as updating the author's name and email in the ``setup.py`` file.

### CODE REVIEW

**Review by abwilf — APPROVED** [review]

**Comment by CristoJV** [comment]
Hi abwilf, please can you update the ``setup.py`` with your metadata such as author's name and email?

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from setuptools import find_packages, setup
+
+requirements_filepath = convert_path("requirements.txt")
+with open(requirements_filepath, "r") as f:
+    requirements = [str(requirement) for requirement in parse_requirements(f)]
+
+setup(
+    name="mmsdk",  # Replace with your package's name
+    version="1.2.0",  # Your package's version
+    author="Your Name",  # Your name
```
Comment (CristoJV) [inline setup.py:13]: Change the author name 😄

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    author_email="your.email@example.com",  # Your email
```
Comment (CristoJV) [inline setup.py:14]: Change the author email 😄

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+from pkg_resources import parse_requirements
+from setuptools import find_packages, setup
+
+requirements_filepath = convert_path("requirements.txt")
+with open(requirements_filepath, "r") as f:
+    requirements = [str(requirement) for requirement in parse_requirements(f)]
+
+setup(
+    name="mmsdk",  # Replace with your package's name
+    version="1.2.0",  # Your package's version
```
Comment (CristoJV) [inline setup.py:12]: Verify the mmsdk version

**Comment by arin688** [comment]
Please help me out for my thesis by giving the CMU-mosei datasets

**Pull request merged**